{
  "underlyingError": "Error",
  "report": "User not found",
  "trace_id": "asdafsdgfs",
  "span_id": "mvssdvnkdslvnkdsl"
}